app/services/permissions_service.rb

```diff
@@ -43,6 +46,7 @@ def guest_permissions
       return true if controller == "skills" && action == "show"
       return true if controller == "professionals" && action == "index"
       return true if controller == "sessions" && action.in?(%w(new create))
+      # a registered user should not be authorized for line 50
       return true if controller == "users" && action.in?(%w(new create))
```

Comment:
I'd like to talk about this a bit more. I'm not sure that I agree that registered users should not be able to create additional accounts. @alishersadikov